feature: update Trieve search and AI Chat to newest version.

cdxker opened the pull request:
## V2 Features have arrived 🚀🚀

Preview is at https://signoz.trieve.ai

- styling: move chat toggle to top right corner as a switch similar to InKeep's component
- styling: make chat look more like InKeep's with input field at bottom instead of top
- feature: add clear button to chat
- feature: add feedback button to chat

cdxker committed:
feat: update Trieve search and AI Chat to the newest version

pranay01 commented on components/SearchButton.tsx:
```diff
@@ -18,6 +18,19 @@ const SearchButton = () => {
+          "How do I install SigNoz?",
+        ]}
+        brandColor="#E75536"
+        brandName='Signoz'
```
Can we change brandname to 'SigNoz'?

cdxker replied:
Fixed!

cdxker commented:
Just fixed the merge conflict and updated the `yarn.lock` for the right `trieve-ts-sdk` version. Let me know if anything else is needed for this to be merged.
